examples/petsc/README.md

```diff
@@ -52,19 +52,24 @@ The following arguments can be specified for all of the above examples:
 - `-test`              - Testing mode (do not print unless error is large)
 - `-benchmark`         - Benchmarking mode (prints benchmark statistics)
 
-### libCEED example to compute surface area using DMPlex - areaplex
+### libCEED examples to compute surface area using DMPlex - areacube and areasphere
 
-This example uses the mass matrix to compute the surface area of a cube, defined via DMPlex. 
+These examples use the mass matrix to compute the surface area of a cube or a discrete cubed-sphere, defined via DMPlex.
 
 To build, run `make area`
 
-To run, `./area -ceed [ceed-resource] -petscspace_degree [degree]`
+To run, `./area -problem cube -ceed [ceed-resource] -petscspace_degree [degree]`
+
+or
+
+`./area -problem sphere -ceed [ceed-resource] -petscspace_degree [degree]`
 
 ### Command line arguments
```

Comment:
If this is meant to be specific to `area`, it should be `####` (nested one level deeper), otherwise remove `-problem` below because it's only relevant to `area`.